.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -29,7 +29,7 @@ else
 fi
 
 # Version qualifier shouldn't be used in PR builds
-if [[ x"$BUILDKITE_PULL_REQUEST" != xfalse && -n "$VERSION_QUALIFIER" ]] ; then
+if [[ x"$BUILDKITE_PULL_REQUEST" != xfalse && -n "${VERSION_QUALIFIER:=""}" ]] ; then
```

Comment:
The `:=""` in `${VERSION_QUALIFIER:=""}` seems redundant, as `bash` will expand `${VERSION_QUALIFIER}` as an empty string if the environment variable is not set.

Reply:
The script calls `set -euo pipefail` on line 11. The `-u` option treats unset variables as errors, so in this case explicitly setting `VERSION_QUALIFIER` to the empty string if it's not set is necessary.